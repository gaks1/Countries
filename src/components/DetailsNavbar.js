import BackButton from './BackButton';
import styles from './DetailsNavbar.module.css';

const DetailsNavbar = () => (
  <nav className={styles.navbar}>
    <div className={styles.divbutton}>
      <BackButton />
    </div>
    <div>
      <h1 className={styles.h1title}>DETAILS</h1>
    </div>
  </nav>
);

export default DetailsNavbar;
