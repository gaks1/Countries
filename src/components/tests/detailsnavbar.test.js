import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import {
  BrowserRouter as Router,
} from 'react-router-dom';

import DetailsNavbar from '../DetailsNavbar';

describe('DetailsNavbar',
  () => {
    test('renders without crashing', () => {
      const { getByText } = render(
        <Router>
          <DetailsNavbar />
        </Router>,
      );
      expect(getByText('DETAILS')).toBeInTheDocument();
    });

    test('matches snapshot', () => {
      const tree = renderer
        .create(
          <Router>
            <DetailsNavbar />
          </Router>,
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
