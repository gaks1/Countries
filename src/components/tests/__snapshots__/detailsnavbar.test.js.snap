// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsNavbar matches snapshot 1`] = `
<nav
  className="navbar"
>
  <div
    className="divbutton"
  >
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      <span
        className="material-symbols-outlined"
      >
        arrow_back_ios
      </span>
    </button>
  </div>
  <div>
    <h1
      className="h1title"
    >
      DETAILS
    </h1>
  </div>
</nav>
`;
